### PR TITLE
Feature/event card style

### PR DESCRIPTION
**Pull request by mdashko**

### CODE REVIEW

File: src/app/main/component/events/services/events.service.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ export class EventsService implements OnDestroy {
     );
   }
 
+  public getUserFavoriteEvents(): Observable<EventResponseDto> {
+    return;
```
Comment (hnativlyubomyr) [inline src/app/main/component/events/services/events.service.ts]: what is the point of this method?) remove it

File: src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts [hunk 2]
```diff
@@ -108,31 +114,45 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
...
+    } else {
       this.eventType = '';
     }
 
     this.initGetUserEvents(this.eventType);
   }
 
+  public escapeFromFavorites(): void {
+    this.isFavoriteBtnClicked = !this.isFavoriteBtnClicked;
+    console.log('go out');
```
Comment (hnativlyubomyr) [inline src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts]: remove console.log

File: src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts [hunk 3]
```diff
@@ -108,31 +114,45 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
...
+  }
+
+  public goToFavorites(): void {
+    this.isFavoriteBtnClicked = true;
+    console.log('here');
```
Comment (hnativlyubomyr) [inline src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts]: the same

File: src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts [hunk 4]
```diff
@@ -108,31 +114,45 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
...
       .getAllUserEvents(0, this.eventsPerPage, this.userLatitude, this.userLongitude, eventType)
       .pipe(take(1))
       .subscribe((res: EventResponseDto) => {
         this.eventsList = res.page;
-        this.eventsTotal = res.totalElements;
+        this.totalEvents = res.totalElements;
       });
   }
 
+  getUserFavouriteEvents(eventType?: string): void {
```
Comment (hnativlyubomyr) [inline src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts]: remove this method

**Review by KKatiaa — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**